src/DocumentDB.ChangeFeedProcessor/LeaseManagement/DocumentServiceLease.cs

```diff
@@ -14,6 +14,9 @@ namespace Microsoft.Azure.Documents.ChangeFeedProcessor.LeaseManagement
     [Serializable]
     internal class DocumentServiceLease : ILease, ILeaseAcquireReasonProvider
     {
+        internal const string IdPropertyName = "id";
+        internal const string LeasePartitionKeyPropertyName = "leasepk";
```

Comment:
it's already a lease. Can we have this as "pk" or "partitionKey", I would prefer the latter as it more clear?